#!/usr/bin/env python
PACKAGE = "franka_example_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("desired_force", double_t, 0,
        "Desired force in the z axis", 0.0, 0.0, 10.0)

gen.add("k_pf", double_t, 0, "Force P gain", 0.0, 0.0, 5.0)

gen.add("v_x", double_t, 0, "x velocity", 0.0, -0.1, 0.1)
gen.add("v_y", double_t, 0, "y velocity", 0.0, -0.1, 0.1)

exit(gen.generate(PACKAGE, "dynamic_hybrid_params", "hybrid_param"))